packages/@okta/vuepress-site/docs/reference/api/schemas/index.md

```diff
@@ -1037,6 +1037,504 @@ curl -v -X POST \
+
+##### Request parameters
+
+
+| Parameter   | Description                                                    | Param Type | DataType                                                          | Required |
+|:------------|:---------------------------------------------------------------|:-----------|:------------------------------------------------------------------|:---------|
+| definitions | Subschema with one or more custom Profile properties to remove | Body       | [Group Profile custom subschema](#group-profile-custom-subschema) | TRUE     |
+
+> **Note:** You must set properties explicitly to `null` to remove them from the Schema, as `POST` is interpreted as a partial update.
+
+##### Response parameters
+
+[Group Schema](#group-schema-object)
+
+##### Request example
+
+
+```bash
+curl -v -X POST \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+-d '{
+  "definitions": {
+    "custom": {
+      "id": "#custom",
+      "type": "object",
+      "properties": {
+        "groupContact": null
+      },
+      "required": []
+    }
+  }
+}' "https://${yourOktaDomain}/api/v1/meta/schemas/group/default"
+```
+
+##### Response example
+
+The following response is only a subset of properties for brevity.
+
+```json
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "_links": {
+    "self": {
+      "href": "http://${yourOktaDomain}/api/v1/meta/schemas/group/default",
+      "method": "GET",
+      "rel": "self"
+    }
+  },
+  "created": "2021-01-30T00:18:24.000Z",
+  "definitions": {
+    "base": {
+      "id": "#base",
+      "properties": {},
+      "required": [
+        "name"
+      ],
+      "type": "object"
+    },
+    "custom": {
+      "id": "#custom",
+      "properties": {
+        "remainingOtherCustomProperty": {
+          "description": "A property that was not deleted",
+          "master": {
+            "type": "PROFILE_MASTER"
+          },
+          "mutability": "READ_WRITE",
+          "permissions": [
+            {
+              "action": "READ_WRITE",
+              "principal": "SELF"
+            }
+          ],
+          "scope": "NONE",
+          "title": "Lorem ipsum",
+          "type": "string"
+        }
+      },
+      "required": [],
+      "type": "object"
+    }
+  },
+  "description": "Okta group profile template",
+  "id": "https://${yourOktaDomain}/meta/schemas/group/default",
+  "lastUpdated": "2021-02-25T23:05:31.000Z",
+  "name": "group",
+  "properties": {
+    "profile": {
+      "allOf": [
+        {
+          "$ref": "#/definitions/custom"
+        },
+        {
+          "$ref": "#/definitions/base"
+        }
+      ]
+    }
+  },
+  "title": "Okta group",
+  "type": "object"
+}
+```
+
 ## User Schema object
 
 The [User object](/docs/reference/api/users/#user-object) schema is defined using [JSON Schema Draft 4](https://tools.ietf.org/html/draft-zyp-json-schema-04).
```

Comment:
Is it User object schema or User Schema object (as the title suggests)?

Reply:
I think the first link is just a link to the user object (users/), but the section talks about the schema for that user object.